inlong-tubemq/tubemq-server/src/main/java/org/apache/inlong/tubemq/server/master/web/MasterStatusCheckFilter.java

```diff
@@ -47,8 +52,8 @@ public void init(FilterConfig filterConfig) throws ServletException {
 
     @Override
     public void doFilter(ServletRequest request,
```

Comment:
This place needs to stay the same

The address provided by the Master when redirecting is a limited set, and it is internal access, so it has little impact if it is not checked.